nucypher/cli/commands/cloudworkers.py

```diff
@@ -67,33 +71,89 @@ def up(general_config, staker_options, config_file, cloudprovider, aws_profile,
     STAKEHOLDER = staker_options.create_character(emitter, config_file)
 
     stakers = STAKEHOLDER.get_stakers()
-    if not stakers:
+    if not stakers and not create_unstaked:
```

Comment:
The output below, "No staking accounts found", no longer makes sense given this conditional.